website/blog/2022-05-19-GAAD-2022-update.md

```diff
@@ -0,0 +1,48 @@
+---
+title: React Native Accessibility - GAAD 2022 Update
+author: Alex Tait
+authorTitle: React Native Accessibility Community Manager
+authorURL: 'https://twitter.com/AT_Fresh_Dev'
+authorImageURL: 'https://github.com/alextait1.png'
+authorTwitter: AT_Fresh_Dev
+tags: [announcement]
+date: 2022-05-19
+---
+
+May 19th, 2022 marks the 11th annual celebration of Global Accessibility Awareness Day and we wanted to update everyone on the accessibility progress we’ve made on the React Native Framework. Meta (formerly Facebook) was the [first organization to take the GAAD pledge in 2020](/blog/2021/03/08/GAAD-React-Native-Accessibility), committing to making the React Native framework accessible.
+
+> _“We hope this pledge makes it easier for developers using React Native to create fully accessible mobile apps and inspires other organizations to make similar commitments to a more accessible future.”_
+>
+> — [Mike Shebanek, head of accessibility Meta, 2020](https://gaad.foundation/gaadpledge/)
+
+The process initially began with a thorough review and gap analysis of the framework focused on React Native utilized the iOS and Android APIs to support accessibility features. Dozens of issues have since been fixed or closed out, making good on the pledge to make React Native accessible and advancing the accessibility of the framework ever forward.
+
+We didn’t stop there, and in early 2022, we reviewed and prioritized the remaining issues from this gap analysis on the [Improved React Native Accessibility Board](https://github.com/facebook/react-native/projects/15) based on their impact on developers and end users.
+
```

Comment:
```suggestion

<!--truncate-->
```
Can we truncate the blog post somewhere? Don't need to be at the suggested line. 🙂

Reply:
Make sense here 👍